python/doc/examples/calibration/bayesian_calibration/plot_gibbs_simus.py

```diff
@@ -552,19 +553,18 @@ def log_cond_tau_post(tau):
 ks = ot.KernelSmoothing()
 posterior = ks.build(post_sample)
 
-# Optional: change the bandwidth
-bandwidth = ks.getBandwidth()
-for k in range(len(bandwidth)):
-    bandwidth[k] = 1.0 * bandwidth[k]
-posterior = ks.build(post_sample, bandwidth)
-
 grid = ot.GridLayout(1, 3)
 grid.setTitle("Bayesian inference")
 xlabs = [r"$\theta_0$", r"$\theta_1$", r"$\tau$"]
+final_value = sample[-10:-1].computeMean()
 for parameter_index in range(3):
     graph = posterior.getMarginal(parameter_index).drawPDF()
-    graph.setColors(ot.Drawable.BuildDefaultPalette(2))
-    graph.setLegends(["Posterior"])
+    prior_pdf = prior.getMarginal(parameter_index).drawPDF()
+    graph.add(prior_pdf)
+    final_v = ot.Curve([final_value[parameter_index]] * 2, [0.0, 1.0])
+    graph.add(final_v)
+    graph.setColors(ot.Drawable.BuildDefaultPalette(3))
+    graph.setLegends(["Posterior", "Prior", "Final value"])
     graph.setXTitle(xlabs[parameter_index])
     grid.setGraph(0, parameter_index, graph)
 _ = View(grid)
```

Comment:
There is an inconsistency in the graphs. In the PDFs of the marginals, we have theta_0, theta_1 and tau, while in the scatter plots we have theta_1, theta_2 and tau.

Reply:
I think you should stick to theta_0 and theta_1, because it fits the Python index convention.